### ISSUE TITLE
GPU memory required for training

### ISSUE THREAD

**Issue by qiuzidian** [chengzhag/PanFusion#10]
Hello! Can I train it on a 24GB 3090?

**Comment by Orange-Ctrl** [comment]
Hi, were you able to successfully run the code on an 3090?

**Comment by qiuzidian** [comment]
no, according to the wandb log, it seems that need 48G to train

**Comment by Orange-Ctrl** [comment]
> no, according to the wandb log, it seems that need 48G to train

I run on 8 24G gpus  and failed.  I have no ideas what went wrong.

**Comment by qiuzidian** [comment]
each GPU's memory should not be less than 40G

**Comment by Orange-Ctrl** [comment]
> each GPU's memory should not be less than 40G

got it, thank you!
